compiler/pipes/calc-bad-vars.cpp

```diff
@@ -591,7 +591,25 @@ class CalcBadVars {
         func->wait_prev = to_parents[func];
       }
     }
+
+    auto get_call_resumable_path = [&](FunctionPtr foo) {
```

Comment:
why `foo`? o_O
IMHO, but `func` is better, but in general `function`

Reply:
Fixed